server/src/internalClusterTest/java/org/opensearch/snapshots/SearchableSnapshotIT.java

```diff
@@ -202,6 +220,47 @@ public void testSearchableSnapshotIndexIsReadOnly() throws Exception {
         );
     }
 
+    public void testDeleteSearchableSnapshotBackingIndexThrowsException() throws Exception {
+        disableRepoConsistencyCheck("reason");
+        final String indexName = "test-index";
+        final Client client = client();
+        final String repoName = "test-repo";
+        final String snapshotName = "test-snap";
+        final StepListener<AcknowledgedResponse> deleteSnapshotStepListener = new StepListener<>();
```

Comment:
Is this used? Same question for line 249.

Reply:
Removed